contrib/src/test/java/com/datatorrent/contrib/hbase/HBasePOJOInputOperatorTest.java

```diff
@@ -59,17 +60,31 @@ public MyGenerator()
       this.setTupleType( TestPOJO.class );
     }
   }
-  
+
+  public static class TestHBasePOJOInputOperator extends HBasePOJOInputOperator
+  {
+    @Override
+    public void setup(OperatorContext context)
+    {
+      try {
```

Comment:
This was added to let the output operator first insert data into hbase table in order for the input operator to read it. Will add comments.